### PR TITLE
CherryPicked master PR 'More Filtered robotello API tests for upgrade #5173'

### PR DESCRIPTION
**Pull request by jyejare**
$subject

### CODE REVIEW

File: tests/foreman/api/test_subscription.py [hunk 1]
```diff
@@ -22,7 +22,9 @@
 from nailgun.entity_mixins import TaskFailedError
 from robottelo.api.utils import upload_manifest
 from robottelo import manifests
-from robottelo.decorators import run_in_one_thread, skip_if_not_set, tier1
+from robottelo.decorators import (
```
Comment (san7ket) [inline tests/foreman/api/test_subscription.py]:
```
from robottelo.decorators import (
run_in_one_thread, 
skip_if_not_set, 
tier1, 
upgrade
)
```
)

Comment (jyejare) [inline tests/foreman/api/test_subscription.py]: Fixed

**Comment by jyejare** [comment]
@san7ket , I fixed ur comment please revisit.

**Comment by jyejare** [comment]
@SatelliteQE/quality-engineers , Guys please review and help me to merge this PR asap

**Pull request merged**